### PR TITLE
Lowercase TRUE/FALSE/NULL in the manual

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: entities/global.ent [hunk 1]
```diff
@@ -651,11 +651,11 @@
...
 <!-- The three special language constants -->
-<!ENTITY true  '<constant xmlns="http://docbook.org/ns/docbook">TRUE</constant>'>
-<!ENTITY false '<constant xmlns="http://docbook.org/ns/docbook">FALSE</constant>'>
-<!ENTITY null  '<constant xmlns="http://docbook.org/ns/docbook">NULL</constant>'>
-<!-- use <type>NULL</type> if you're talking about the _type_ NULL,
-     use &null; if it's about the _value_ NULL
+<!ENTITY true  '<constant xmlns="http://docbook.org/ns/docbook">true</constant>'>
+<!ENTITY false '<constant xmlns="http://docbook.org/ns/docbook">false</constant>'>
+<!ENTITY null  '<constant xmlns="http://docbook.org/ns/docbook">null</constant>'>
+<!-- use <type>null</type> if you're talking about the _type_ null,
```
Comment (kocsismate) [inline entities/global.ent:657]: In the sense of this comment, I believe I'll have to modify the methodsynopsis generator not to generate `&null` as part of an union type

Comment (cmb69) [reply entities/global.ent:657]: Well, IMO `<type>null</type>` is good enough.

**Review by cmb69 — APPROVED** [review]
Thanks! Good for me.

Does anybody object?

**Comment by salathe** [comment]
> Does anybody object?

My thoughts are as they were back in [December 2016](https://news-web.php.net/php.doc/969386410):

> I have no strong thoughts either way and it seems like change for change's
sake. I like the distinction between NULL the value and null the type, but
that's about all that's going for the existing convention..

**Review by nikic — APPROVED** [review]

**Pull request closed without merging**